internal/provider/schema_cloudflare_access_identity_provider.go

```diff
@@ -130,6 +130,10 @@ func resourceCloudflareAccessIdentityProviderSchema() map[string]*schema.Schema
 						Type:     schema.TypeString,
 						Optional: true,
 					},
+					"pkce_enabled": {
```

Comment:
should we default this to `false`?

Reply:
Don't think we need to. This var has the `omitempty` tag in the go struct so the API just won't populate it.